keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ImageProvider.kt

```diff
@@ -29,14 +29,6 @@ import com.netflix.spinnaker.keel.ec2.jackson.ImageProviderDeserializer
 @JsonDeserialize(using = ImageProviderDeserializer::class)
 sealed class ImageProvider
 
-/**
- * Provides the ami id for the image
- */
-@JsonDeserialize(using = JsonDeserializer.None::class)
-data class IdImageProvider(
-  val imageId: String
-) : ImageProvider()
```

Comment:
No longer used since this is not compatible with multi region clusters. Instead image id can be specified directly in each region if this is what you want.